sdk/src/main/java/com/meniga/sdk/webservices/MenigaAPI.java

```diff
@@ -123,6 +124,9 @@
 	@DELETE(APIConst.URL_TRANSACTIONS + "/{id}/" + APIConst.COMMENTS + "/{commentId}")
 	Call<Void> deleteComment(@Path("id") long transactionId, @Path("commentId") long commentId);
 
+	@POST(APIConst.URL_TRANSACTIONS + "/" + APIConst.COMMENTS)
```

Comment:
You can check this commit 3b81ac6 to see how it can be refactored to avoid bloating this class.